geodiff/src/tableschema.cpp

```diff
@@ -110,7 +110,7 @@ TableColumnType postgresToBaseColumn(
   {
     type.baseType = TableColumnType::INTEGER;
   }
-  else if ( dbType == "double precision" || dbType == "real" )
+  else if ( dbType == "double precision" || dbType == "real" || startsWith( dbType, "numeric" ) )
```

Comment:
Shall we list `decimal` here as well if it is listed in postgresdriver.cpp?  (it's a bit annoying that this code is spread across two different places, maybe this whole function should be in the postgres driver, but that's probably for a separate refactoring PR)

Reply:
Oh, you are right. My fault.